datahub-web-react/src/images/datahub-logo-color-loading_pendulum.svg

```diff
@@ -15,49 +15,53 @@
         </linearGradient>
 
         <mask id="red_mask">
-            <circle cx="90" cy="90" r="34" fill="white" />
-            <circle cx="90" cy="90" r="26" fill="black" />
-            <rect width="34" height="68" x="56" y="56" fill="black" />
-            <circle cx="90" cy="60" r="4" fill="white" />
-            <circle cx="90" cy="120" r="4" fill="white" />
+            <circle r="34" fill="white" />
```

Comment:
This one is my favorite :)

Reply:
How about making a watch with a red arc for the second hand, a yellow arc for the minute hand, and a blue arc for the hour hand? ⌚ 
DataHub logo would point to 1:40. 😄

Reply:
like this 🕐 
![datahub-logo-color-loading_clock](https://user-images.githubusercontent.com/26197162/159927381-1cf11f15-44f4-4945-bb9c-32ded6aa368e.svg)